includes/bookmark.inc

```diff
@@ -841,6 +841,10 @@ abstract class Bookmark implements BookmarkInterface {
         $params[] = $form_state['values']['bookmarks']['fieldset']['format_opts']['export_styles'][$key];
```

Comment:
If you are passing $form_state complete now, then this conditional should not be needed? $params do not have associative keys, so the value depends the position?